### PR TITLE
Support for save/restore system clock from file

### PR DESCRIPTION
**Pull request by troglobit**
## Description

This PR mainly addresses issues with malfunctioning RTC devices.  A few minor "infra" changes have also been included, listed at the end:

Major:

 - Disable RTC on Styx boards, can return time in the future, and no battery backup
 - Backport [upstream Finit patches](https://github.com/troglobit/finit/pull/419) to support saving and restoring system clock from a file
 - Enable Finit RTC fallback to disk file and set fixed RTC fallback value in all defconfigs
 - Fix finit.mk handling of .config string values

Minor:

 - `mnt` script, add missing `-n` to test
 - Hide "use persistent /var ..." section in `qemu.sh -c` when disk image is selected
 - Prevent disk image from being recreated every time `qemu.sh` is started
 - Fix misaligned grub menu when booting disk image in `qemu.sh`
 - Add support for tweaking emulated RTC behavior in `qemu.sh`

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [X] Bugfix
  - [ ] Regression tests
  - [x] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Test changes
  - [ ] Checked in changed Readme.adoc (make test-spec)
  - [ ] Added new test to group Readme.adoc and yaml file
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
  - [ ] ChangeLog updated (for major changes)
- [ ] Other (please describe):

### CODE REVIEW

**Comment by troglobit** [comment]
@wkz pretesting Finit patches, I'll make sure to add a ChangeLog update and a better PR description tomorrow.

**Comment by troglobit** [comment]
@mattiaswal maybe you can focus your review on the non RTC-related changes? (If you're pressed for time, that is.)

**Review by mattiaswal — APPROVED** [review]
LGTM :rocket:

**Review by wkz — APPROVED** [review]
:tada:

File: board/common/rootfs/usr/libexec/infix/mnt [hunk 1]
```diff
@@ -146,7 +146,7 @@ mount_overlay home "$cfgsrc" /home
 mount_overlay root "$cfgsrc" /root
 mount_bind         "$varsrc" /var
 
-[ "$vlibsrc" ] && mount_bind "$vlibsrc" /var/lib
+[ -n "$vlibsrc" ] && mount_bind "$vlibsrc" /var/lib
```
[the remark this answers is not in the document]
Comment (troglobit) [reply board/common/rootfs/usr/libexec/infix/mnt:149]: Ah, did not know this.  Was a `shellcheck` comment I just noticed when debugging an issue that turned out to be caused by the disk image being recreated every time `qemu.sh` was called.

**Pull request merged**